osu.Game/Screens/Multi/Lounge/Components/RoomsContainer.cs

```diff
@@ -88,6 +97,14 @@ private void addRooms(IEnumerable<Room> rooms)
             {
                 roomFlow.Add(new DrawableRoom(room)
                 {
+                    DuplicateRoom = () =>
+                    {
+                        Room newRoom = room.CreateCopy();
+                        if (!newRoom.Name.Value.StartsWith("Copy of "))
+                            newRoom.Name.Value = $"Copy of {room.Name.Value}";
```

Comment:
this should be inside `CreateCopy`

Reply:
Also this shouldn't be "Copy of ...". This should be the "{user}'s awesome room" text.

Reply:
For the time being I'm going to copy the name from the original room. It's up to the user to change this if they want to, in my opinion.